src/status_im/chat/models/message_test.cljs

```diff
@@ -1,148 +0,0 @@
-(ns status-im.chat.models.message-test
```

Comment:
Why are we removing this test file? Shouldn't we adapt it?

Reply:
mhm good question

Reply:
removed it by mistake, thanks. reverted

Reply:
Thanks Andrey